pytorch_lightning/trainer/trainer.py

```diff
@@ -747,7 +747,7 @@ def __set_random_port(self):
             default_port = random.randint(10000, 19000)
             os.environ['MASTER_PORT'] = str(default_port)
 
-    def __attach_dataloaders(self, model, train_dataloader, val_dataloaders, test_dataloaders):
+    def __attach_dataloaders(self, model, train_dataloader=None, val_dataloaders=None, test_dataloaders=None):
```

Comment:
.fit doesn't need a test loader... we should have removed this in the first place haha. please remove here as well (from .fit(...))

Reply:
Done.